src/v/raft/consensus.cc

```diff
@@ -1053,7 +1053,7 @@ ss::future<> consensus::do_start() {
                   _term = lstats.dirty_offset_term;
                   _voted_for = {};
               }
-
+              _flushed_offset = lstats.dirty_offset;
```

Comment:
i realize this is in start-up so it's a special case, but can you add a comment to explain it. `flushed = dirty` alone without a comment _sounds_ incorrect.